amundsen_application/static/js/components/TableDetail/RequestMetadataForm/constants.ts

```diff
@@ -0,0 +1,9 @@
+/* TODO: harcoded string that should be translatable/customizable */
+export const TITLE_TEXT = 'Amundsen Resource Request';
+export const FROM = 'From';
+export const TO = 'To';
```

Comment:
This variable name is probably too short. `FROM_LABEL` and `TO_LABEL` might be less ambiguous. That being said, it's also possible this is overkill.